src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -179,13 +196,32 @@ static Builder builder() {
         Builder maxUses(int maxUses);
 
         /**
-         * Sets the trade offer to be generated to grant experience upon use.
+         * Sets the trade offer to be generated to grant experience to the customer upon use.
          *
          * @param experience Whether the offer will grant experience
          * @return This builder
          */
         Builder canGrantExperience(boolean experience);
 
+        /**
+         * Sets how much experience the trade offer will grant to the merchant upon use.
+         *
+         * @param experience The amount of experience.
+         * @return This builder
+         */
+        Builder merchantExperience(int experience);
+
+        /**
+         * Sets the rate at which this trade offer's price will grow when demand for an item increases.
```

Comment:
Maybe there should be a hint what this rate does. Does every trade increase it? Is this rate incremented by this value or multiplied by it? How does the user pay 1.05 emeralds?